src-built-in/components/toolbar/stores/toolbarStore.js

```diff
@@ -251,25 +244,69 @@ class _ToolbarStore {
 	 *
 	 */
 
-	setupHotkeys(cb) {
-		var self = this;
-		if (storeOwner) {
-			let keys = FSBL.Clients.HotkeyClient.keyMap;
-			FSBL.Clients.HotkeyClient.addGlobalHotkey([keys.ctrl, keys.alt, keys.up], () => {
+
+	/**
+ * Load the hotkeys from the config.json. If there are no items in config.json, hotkeys use the defaults in the current file.
+ *
+ *
+ * @param {any} done
+ * @memberof _ToolbarStore
+ */
+	loadHotkeysFromConfig(done) {
+
+		// destructure toolbarConfig.hotkeys and provide a default fallback value for each hotkey.
+		const {
+			showToolbar = ["ctrl", "alt", "t"],
+			hideToolbar = ["ctrl", "alt", "h"],
+			bringWindowsToFront = ["ctrl", "alt", "up"],
+			minimizeAll = ["ctrl", "alt", "down"],
+			showSearch = ["ctrl", "alt", "f"],
+		} = toolbarConfig.hotkeys
+
+		// show the toolbar
+		FSBL.Clients.HotkeyClient.addGlobalHotkey(
+			showToolbar,
+			() => {
+				this.Store.setValue({ field: "searchActive", value: false });
+				this.showToolbarAtFront();
+			});
+
+		// hide the toolbar
+		FSBL.Clients.HotkeyClient.addGlobalHotkey(
+			hideToolbar,
+			() => {
```

Comment:
Is the anonymous function necessary here? Can't it just be:
```
FSBL.Clients.HotkeyClient.addGlobalHotkey(hideToolbar, this.hideToolbar);
```
I think this is true for all of them except showToolbar and showSearch.